packages/web-pkg/src/components/AppBar/AppBar.vue

```diff
@@ -207,6 +210,14 @@ export default defineComponent({
         ] as FileAction[]
       }
 
+      const actionExtensions = requestExtensions<ActionExtension>({
+        id: 'app.files.batch-actions',
```

Comment:
Hmm - it's a bit weird to reference an extension point defined in an app in web-pkg ... 👀

Reply:
agreed. as discussed in chat: we'll tackle that in a followup.